tapeout/src/test/resources/macros/rocketchip.json

```diff
@@ -0,0 +1,76 @@
+[
+  {
+    "type": "sram",
+    "name": "tag_array_ext",
+    "depth": 64,
+    "width": 80,
+    "ports": [
+      {
+        "clock port name": "RW0_clk",
+        "mask granularity": 20,
+        "output port name": "RW0_rdata",
+        "input port name": "RW0_wdata",
+        "address port name": "RW0_addr",
+        "mask port name": "RW0_wmask",
+        "chip enable port name": "RW0_en",
+        "write enable port name": "RW0_wmode"
+      }
+    ]
+  },
+  {
+    "type": "sram",
+    "name": "T_1090_ext",
```

Comment:
I'm a little confused about what this file represents? Why should it have chisel generated temporary names for the srams?